README.md

```diff
@@ -7,40 +7,89 @@
 <!-- toc -->
 
 - [Overview](#overview)
-- [Status](#status)
 - [Projects](#projects)
   - [Model Signing](#model-signing)
+  - [SLSA for ML](#slsa-for-ml)
+- [Status](#status)
 - [Contributing](#contributing)
 
 <!-- tocstop -->
 
 ## Overview
 
-This repository will host a collection of utilities and examples related to the security of machine learning pipelines. The focus is on providing *verifiable* claims about the integrity and provenance of the resulting models, meaning users can check for themselves that these claims are true rather than having to just trust the model trainer.
+There is a significant growth in the number of ML-powered applications. However,
+this also provides grounds for attackers to exploit unsuspecting ML users. This
+is why Google launched [Secure AI Framework (SAIF)][saif] to help chart a path
+towards creating trustworhty AI applications. The first principle of SAIF is
 
-## Status
+> Expand strong security foundations to the AI ecosystem
 
-This is not an officially supported Google product.
+Building on the work with [Open Source Security Foundation][openssf], we are
+creating this repository to demonstrate how the ML supply chain can be strengthened in
+_the same way_ as the traditional software supply chain.
 
-This project is currently in alpha. We may make breaking changes until the first official release. All code should be viewed as experimental and should not be used in any production environment.
+This repository hosts a collection of utilities and examples related to the
+security of machine learning pipelines. The focus is on providing *verifiable*
+claims about the integrity and provenance of the resulting models, meaning users
+can check for themselves that these claims are true rather than having to just
+trust the model trainer.
 
 ## Projects
 
+Currently, there are 2 main projects in the repository: model signing (to
```

Comment:
Nit: change '2' to 'two' (best to spell out numbers under 10).

Reply:
Sounds good, changed!